src/content/guides/creating-token-collections.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Creating Token Collections
+description: Create token collections, define redemption conditions, and mint tokens for the collections.
+nav:
+  path: Reference/Digital Assets
+  order: 3
+---
+
+## What is a Digital Token?
+
+Digital tokens are assets that can be exchanged for specific products or deals. Think a "Free coffee" coupon or a "50% off" voucher.
+Unlike assets such as gift cards or currency, tokens can only be spent in their entirety, all at once.
+
+### Token Collection
+
+A collection is a grouping of tokens that share the same attributes and redemption conditions, e.g.
+
+- Name
+- Icon
+- Expiry period
+- Maximum redemption value
+
+### Redemption Conditions
+
+A redemption condition allows a merchant to accept tokens from a token collection.
+It contains redemption details specific to that merchant, e.g. redeemable product SKUs.
+
+
+## Implementation
+
+``` mermaid
+sequenceDiagram
+  autonumber
+  Token Issuer ->> Centrapay: Create Collection
+  Note left of Token Issuer: Confirm redeemable product<br/>IDs with merchant
+  Token Issuer ->> Centrapay: Create Redemption Condition
+  Token Issuer ->> Centrapay: Create Token
+  Token Issuer ->> Centrapay: Transfer Token
+```
+
+1. **Create a token collection**<br/>You can create Collections by calling our [Create Token Collection](https://docs.centrapay.com/api/tokens#create-token-collection-experimental) endpoint. You will need the account identifier of the owning account for the `accountId` field. The `maxValue` field is optional, and defines the upper limit on any settlement amount for a redeemed token.
```

Comment:
@seliwe You don't need the `<br />` tag to achieve the styling you're looking for here. Markdown can do this 👍 

1. **Create a token collection**
  You can create Collections

Reply:
In general, try to avoid using HTML in our content files. It makes them difficult to maintain (we need a better tool than MDX IMO)

Reply:
Ah thanks